bucket/mu-editor.json

```
@@ -0,0 +1,55 @@
+{
+    "homepage": "https://codewith.mu",
+    "version": "1.0.2",
+    "license": "GPL-3.0-or-later",
+    "description": "A small, simple editor for beginner Python programmers.",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/mu-editor/mu/releases/download/1.0.2/mu-editor_1.0.2_win64.exe#/dl.7z",
+            "hash": "84e74aef73b884a9e94092a55e512e9f6e2fd459a573dd4d9d0ed18afe1eba37"
+        },
+        "32bit": {
+            "url": "https://github.com/mu-editor/mu/releases/download/1.0.2/mu-editor_1.0.2_win32.exe#/dl.7z",
+            "hash": "576baea0bd64649583a61bf6ce2836a91b02dce450c4b886d6adc7c35460ae8c"
+        }
+    },
+    "pre_install": [
+        "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninst*\" -Recurse -Force",
+        "if ((Get-Command Get-Content).Parameters.ContainsKey('AsByteStream')) {",
+        "   $enc = @{",
```

You can keep initialization as one line.

```suggestion
        "   $enc = @{ 'AsByteStream' = $true }",
```